README.md

```diff
@@ -1,23 +1,27 @@
 ## Installation
 
-Once QIIME2 is [installed](https://docs.qiime2.org/2017.5/install/native/), it should be possible to install `q2-fragment-insertion` with:
+Once QIIME2 is [installed](https://docs.qiime2.org/2017.10/install/native/), it should be possible to install `q2-fragment-insertion` with:
 
-    source activate qiime2-2017.5
-    git clone https://github.com/wasade/q2-fragment-insertion.git
+    source activate qiime2-2017.10
+    git clone https://github.com/biocore/q2-fragment-insertion.git
     cd q2-fragment-insertion
     python setup.py install
     qiime dev refresh-cache
-    
+
 You will need Java to run SEPP, and you may need to install it. How you do it is dependent on your operating system. If you're on OSX, you likely need to grab a legacy version of Java. Information can be found [here](https://support.apple.com/kb/dl1572?locale=en_US).
 
 ## Important
 
-Currently only insertion into Greengenes 13_8 at 99% is available. We have not exposed mechanisms yet to construct references which can be used, however this can be accomplished by using [SEPP](https://github.com/smirarab/sepp) directly.
+Default reference (phylogeny and matching alignment) is Greengenes 13_8 at 99%.
+You can provide your own reference via optional inputs `--i-reference-alignment` and `--i-reference-phylogeny`. Make sure that every tip of the reference phylogeny has exactly one corresponding sequence in the reference alignment. Taxonomic lineage is obtained from the provided reference phylogeny, which may or may not be decorated with internal node names containing `_` `_`.
```

Comment:
"Taxonomic lineage is..." -> "Insertion taxonomic lineage information can be..."

I don't understand the "__" remark. Does this mean only internal nodes with labels that include Greengenes style prefixes will be considered as lineage information?

Reply:
fixed wording.
If you don't understand it, than my text is really bad. I rephrased. Hopefully it is more clear now.